indra/llwindow/llwindowwin32.cpp

```diff
@@ -5014,6 +4873,47 @@ bool LLWindowWin32::LLWindowWin32Thread::wakeAndDestroy()
         PostMessage(old_handle, WM_DUMMY_, wparam, 0x1337);
     }
 
+    // There are cases where window will refuse to close,
```

Comment:
Now what is this? This got removed in 21947778baaca205615a71a97ac8f563c998fdd3 - now it is back! This entire file is now a huge mess because people decided to work on the same issue in two different branches and now everything is completely mixed and messed up.

Maybe just completely replace this mess with the version from the `develop` branch!

Reply:
Sounds like a plan.

Reply:
Wait for it. There is a lot more stuff wrong...

Reply:
I'll get back to this work tomorrow morning as well.

On the positive side, this should be the last "disaster-level" merge from the era before we switched to git-flow.

Reply:
> On the positive side, this should be the last "disaster-level" merge from the era before we switched to git-flow.

What about DRTVWR-558 (puppetry) and DRTVWR-584 (assimp) then? Have they been discarded?

Reply:
> (puppetry) and (assimp)

No idea, I hadn't worked on them.